<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModalView</class>
 <widget class="QWidget" name="ModalView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="modalBgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,240);</string>
   </property>
  </widget>
  <widget class="QLabel" name="modalInfoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>440</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
              background-color: rgba(0,0,0, 0%)</string>
   </property>
   <property name="text">
    <string>Unlocking Device...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="modalIcon">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>440</width>
     <height>200</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="setPasswordWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>400</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0)</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>442</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLineEdit" name="setDeviceName">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:white;
                              border:0px;
                              border-radius: 5px;
                              padding:4px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>New wallet name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setPasswordOld">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:white;
                              border:0px;
                              border-radius: 5px;
                              padding:4px;</string>
       </property>
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Old password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="setPassword0">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:white;
                              border:0px;
                              border-radius: 5px;
                              padding:4px;</string>
       </property>
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>New password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setPassword1">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:white;
                              border:0px;
                              border-radius: 5px;
                              padding:4px;</string>
       </property>
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Repeat password</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="setPassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#777777;
                              color:white;
                              border:0px;
                              border-radius: 5px;
                              padding:4px;</string>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelSetPassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#777777;
                       color:white;
                       border:0px;
                       border-radius: 5px;
                       padding:4px;
</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="passwordWarning">
         <property name="styleSheet">
          <string notr="true">color: orange;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="passwordInfo2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>* The password also encrypts SD card backups. The same password is therefore REQUIRED when restoring a backup. Don't forget it!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="setpasswordspacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QRCodeSequence" name="qrCodeSequence" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modalInfoLabelLA">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>270</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
              background-color: rgba(0,0,0, 0%)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>450</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>440</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="showDetailsButton">
       <property name="styleSheet">
        <string notr="true">color: white; text-align:left; text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Offline Verification</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>modalBgWidget</zorder>
  <zorder>modalIcon</zorder>
  <zorder>setPasswordWidget</zorder>
  <zorder>qrCodeSequence</zorder>
  <zorder>modalInfoLabelLA</zorder>
  <zorder>modalInfoLabel</zorder>
  <zorder>widget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRCodeSequence</class>
   <extends>QWidget</extends>
   <header>qrcodesequence.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
